src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -26,10 +25,9 @@ export type CardKebabMenuButtonDeps = {
 } & IssueFilingButtonDeps;
 
 export interface CardKebabMenuButtonState {
-    isContextMenuVisible: boolean;
     showNeedsSettingsContent: boolean;
-    target?: HTMLElement | string | MouseEvent | IPoint | null;
     showingCopyToast: boolean;
+    toastText: string;
```

Comment:
I think it'd better if we extract the toast component in a separate and toasText should be a prop, that would help even refactoring with the copy-issue-details **button**

Reply:
Agreed, but that itself could be a separate PR.